webapp/locales/en.json

```diff
@@ -313,26 +313,23 @@
       "successAdd": "Added social media. Updated user profile!",
       "successDelete": "Deleted social media. Updated user profile!"
     },
-    "blocked-users": {
-      "name": "Blocked users",
+    "muted-users": {
+      "name": "Muted users",
       "explanation": {
-        "intro": "If another user has been blocked by you, this is what happens:",
-        "your-perspective": "The blocked person's posts will no longer appear in your news feed.",
-        "their-perspective": "Vice versa: The blocked person will also no longer see your posts in their news feed.",
-        "search": "Posts of blocked people disappear from your search results.",
-        "notifications": "Blocked users will no longer receive notifications if they are mentioned in your posts.",
-        "closing": "This should be sufficient for now so that blocked users can no longer bother you."
+        "intro": "If another user has been muted by you, this is what happens:",
+        "your-perspective": "The muted person's posts will no longer appear in your news feed.",
+        "search": "Posts of muted people disappear from your search results."
       },
       "columns": {
         "name": "Name",
         "slug": "Slug",
-        "unblock": "Unblock"
+        "unmute": "Unmute"
       },
-      "empty": "So far, you have not blocked anybody.",
-      "how-to": "You can block other users on their profile page via the content menu.",
-      "block": "Block user",
-      "unblock": "Unblock user",
-      "unblocked": "{name} is unblocked again"
+      "empty": "So far, you have not muted anybody.",
+      "how-to": "You can mute other users on their profile page via the content menu.",
+      "mute": "Mute user",
+      "unmute": "Unmute user",
+      "unmuted": "{name} is unmuted again"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-16T18:41:36Z" title="Thursday, January 16th 2020, 7:41:36 pm +01:00">Jan 16, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -313,26 +313,26 @@
       "successAdd": "Added social media. Updated user profile!",
       "successDelete": "Deleted social media. Updated user profile!"
     },
-    "blocked-users": {
-      "name": "Blocked users",
+    "muted-users": {
+      "name": "Muted users",
       "explanation": {
-        "intro": "If another user has been blocked by you, this is what happens:",
-        "your-perspective": "The blocked person's posts will no longer appear in your news feed.",
-        "their-perspective": "Vice versa: The blocked person will also no longer see your posts in their news feed.",
-        "search": "Posts of blocked people disappear from your search results.",
-        "notifications": "Blocked users will no longer receive notifications if they are mentioned in your posts.",
-        "closing": "This should be sufficient for now so that blocked users can no longer bother you."
+        "intro": "If another user has been muted by you, this is what happens:",
+        "your-perspective": "The muted person's posts will no longer appear in your news feed.",
+        "their-perspective": "However: The muted person will still be able to see your posts in their news feed.",
+        "search": "Posts of muted people disappear from your search results.",
+        "notifications": "Muted users will still receive notifications if they are mentioned in your posts.",
+        "closing": "This should be sufficient for filtering out content from muted users."
       },
       "columns": {
         "name": "Name",
         "slug": "Slug",
-        "unblock": "Unblock"
+        "unmute": "Unmute"
       },
-      "empty": "So far, you have not blocked anybody.",
-      "how-to": "You can block other users on their profile page via the content menu.",
-      "block": "Block user",
-      "unblock": "Unblock user",
-      "unblocked": "{name} is unblocked again"
+      "empty": "So far, you have not muted anybody.",
+      "how-to": "You can mute other users on their profile page via the content menu.",
+      "mute": "Mute user",
+      "unmute": "Unmute user",
+      "unmuted": "{name} is unmuted again"
```


Are there translations in other languages for this?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-16T19:18:51Z" title="Thursday, January 16th 2020, 8:18:51 pm +01:00">Jan 16, 2020</time>_

---


the translations still need to be fixed
I changed the keys, but the actual translations still say block/unblock